src/metabase/actions/actions.clj

```diff
@@ -54,6 +56,12 @@
      (keyword action)])
   :hierarchy #'driver/hierarchy)
 
+(methodical/defmethod perform-action!* :around :default
+  [action context inputs]
+  (log/tracef "In action %s\nScope: %s\nInvocation stack:%s\nInputs: %s" action (:scope context) (:invocation-stack context) (pr-str inputs))
+  (u/prog1 (next-method action context inputs)
+    (log/tracef "Out action %s: %s" action (pr-str <>))))
```

Comment:
Are you happy with printing the whole `:context` of the result?

Maybe we want to just print the `:outputs`, and maybe some summary statistics about the `:effects`?

Reply:
Just felt asymmetrical with how you suppress logging the initial context, but yeah better to log the whole thing for now than have no visibility on the effects.